### ISSUE TITLE
Simple Pages - Browse - Hierarchy View - Lists Not Announced on Safari

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-SimplePages#46]
A list role needs to be added to the ul in the Simple Pages | Browse | Hierarchy View in order to ensure the list, which uses `list-style:none`, is recognized in the accessibility tree (issue for Safari/VO).

1.3.1 Info and Relationships; Level A; WCAG 2.0
